app/helpers/admin/dashboard_helper.rb

```diff
@@ -7,4 +7,8 @@ def feature_hint(feature, enabled)
 
     safe_join([feature, content_tag(:span, indicator, class: class_names)])
   end
+
+  def short_number_format(number)
+    number_to_human number, units: { unit: '', thousand: 'K', million: 'M', billion: 'B' }
```

Comment:
Why is this necessary? Followers count on profile pages already uses short number format

Reply:
It existed for switching between short and full notation. Match the way you write your profile page.